content/pages/meetup/2023/0920-nycu.rst

```diff
@@ -0,0 +1,100 @@
+===============================================================
+Meetup 2023 September 20th in NYCU (do-unknown-things training)
+===============================================================
+
+:date: 2023-08-31 13:00
+:url: meetup/2023/0920-nycu
+:save_as: meetup/2023/0920-nycu.html
+
+
+Date
+-----
+
+* Date: September 20th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+* 18:30-19:00 Check in and icebreaking
+* 19:00―19:10 Presentation: what are the things
```

Comment:
Please make the schedule separate in 3 sections: checking in, training, and meeting.  The current format is confusing.

Reply:
I have revised the agenda accordingly, but I keep the "Free discussion".